### PR TITLE
Add information about SSH to user docs

### PR DESCRIPTION
**Pull request by MichaelDeciso**

### CODE REVIEW

File: source/manual/how-tos/user-local.rst [hunk 1]
```diff
@@ -50,3 +51,26 @@ page.
...
+Under the "Secure Shell" heading, the following options are available:
+
+============================ =====================================================================
+ **Enable secure shell**      Global on/off switch.
+ **Login Group**              Which user groups can access OPNsense via SSH.
+ **Permit root user login**   Normally, only non-root accounts are allowed for security reasons.
+                              This option enables root login.
+ **Permit password login**    The recommended login method is using SSH keys as it's more secure,
+                              but this option will also enable password logins.
+ **SSH Port**                 Is 22 by default, but can be changed.
```
[the remark this answers is not in the document]
Comment (MichaelDeciso) [reply source/manual/how-tos/user-local.rst]: What are BF bots? I did try looking it up but didn't find anything that looked correct.

File: source/manual/how-tos/user-local.rst [hunk 2]
```diff
@@ -50,3 +51,27 @@ page.
...
+ **Login Group**              Which user groups can access OPNsense via SSH.
+ **Permit root user login**   Normally, only non-root accounts are allowed for security reasons.
+                              This option enables root login.
+ **Permit password login**    The recommended login method is using SSH keys as it's more secure,
+                              but this option will also enable password logins.
+ **SSH Port**                 Defaults to 22, but can be changed to make port scanning less effective.
+ **Listen interfaces**        By default, SSH listens on all interfaces. You can limit this
+                              (to just the LAN, for example) for additional security
+                              at the cost of availability.
+============================ =====================================================================
```
Comment (fabianfrz) [inline source/manual/how-tos/user-local.rst]: it looks like the effective word is out of this bound

File: source/manual/how-tos/user-local.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (fabianfrz) [reply source/manual/how-tos/user-local.rst]: brute force is a technique where you try out all passwords (or a word list) to gain access to a system.

[the remark this answers is not in the document]
Comment (MichaelDeciso) [reply source/manual/how-tos/user-local.rst]: I know what brute force is; I just didn't recognise the abbreviation "BF".

File: source/manual/how-tos/user-local.rst [hunk 3]
```diff
@@ -50,3 +51,27 @@ page.
    :width: 100%
 
 After making the right selection click on **Save** to store the new settings.
+
+SSH and console login
+---------------------
+
+User accounts can be used for logging in to the web frontend, as well as for logging in to the console (via VGA,
+serial or SSH). The latter will only work if the user's shell is not set to ``/sbin/nologin`` and if the user has
+the "shell account access" privilege (see previous paragraph).
```
Comment (AdSchellevis) [inline source/manual/how-tos/user-local.rst]: @MichaelDeciso "shell account access" I don't think this is actually the case, can you check?https://github.com/opnsense/core/commit/5a9f6c4cfb900aa566ecbcaf0018153660f8b681

Comment (MichaelDeciso) [reply source/manual/how-tos/user-local.rst]: You're right. I have modified the text. I also removed the reference to the "shell account access" privilege from the previous chapter.

**Pull request merged**